contrib/jenkins_tests/globals.sh

```diff
@@ -6,6 +6,14 @@ WORKSPACE=${WORKSPACE:=$(pwd)}
 BUILD_NUMBER=${BUILD_NUMBER:=0}
 HOSTNAME=${HOSTNAME:=$(uname -n 2>/dev/null)}
 
+# mounting hugetlbfs for Gtest on xlio_doca: HPCINFRA-1968
```

Comment:
Not sure this should be running always, it is only for gtest, it should be set as a function and be called from gtest script (or whoever else is needs it)

Reply:
Will make it as a function